__mocks__/copy-to-clipboard.js

```diff
@@ -0,0 +1 @@
+export default jest.genMockFromModule('copy-to-clipboard');
```

Comment:
We have to mock the copy-to-clipboard module, since it runs a command that causes all kind of problems for Jest. In any case, we shouldn't be testing 3rd party libraries, so it'll be better to have the copy functionality do nothing.